cpp/tests/link_analysis/hits_test.cpp

```diff
@@ -141,6 +141,12 @@ class Tests_Hits : public ::testing::TestWithParam<std::tuple<Hits_Usecase, inpu
   virtual void SetUp() {}
   virtual void TearDown() {}
 
+  template <typename vertex_t, typename edge_t, typename weight_t>
+  void run_current_test(std::tuple<Hits_Usecase const&, input_usecase_t const&> const& param)
+  {
+    run_current_test<vertex_t, edge_t, weight_t>(std::get<0>(param), std::get<1>(param));
+  }
+
```

Comment:
What's the intention of creating this version of `run_current_test`? If we don't need to keep both, shouldn't we better remove one?

Reply:
The new signature (`std::tuple<...>` instead of sending in the two input arguments) makes things a little cleaner.

I have only applied it to the more expensive tests as part of this PR.  I can certainly combine the two `run_current_test` calls in these tests, it would be a little cleaner.

Reply:
Fixed in latest push.